code-challenges.test.js

```diff
@@ -70,4 +126,26 @@ const accountTransactions3 = []
 
 // b) Create the function that makes the test pass.
 
+//Jest Testing
+// describe("accumulatingSum", () => {
+//   it("Returns the accumulating sum of the array. If an empty array, then it should return an empty array.", () => {
+//     expect(accumulatingSum(accountTransactions1)).toEqual([100, 83, 60, 51])
+//     expect(accumulatingSum(accountTransactions2)).toEqual([250, 161, 261, 165])
+//     expect(accumulatingSum(accountTransactions3)).toEqual([])
+//   })
+// })
+
 // Pseudo code:
+//Function Name: accumulatingSum
+//Input: Array of numbers
+//Output: Array of accumulated sum of the numbers inside.
+//Process: .map() to create a new array with the given information. 
+
+const accumulatingSum = (arr) => {
+    let addSum = 0
+    return arr.map((value) => addSum += value)
+    //Not understanding how addSum += value works. Could you explain it. Please and thank you
+}
+console.log(accumulatingSum(accountTransactions1))
+
+//Feeling a bit disappointed that I couldn't solve these problems by myself. Had to get help from others. 
```

Comment:
There is nothing wrong with that! The point of these assessments is to grow and learn from them. As long as you learned from it, that's a win! I think you're doing really great!